Fields in WebAuthn comments

stevenGravy opened the pull request:
Fields were not marked as required or optional for webauthn initially.

stevenGravy committed:
Fields in WebAuthn comments

alexfornuto commented on docs/pages/access-controls/guides/webauthn.mdx:
```diff
@@ -41,9 +41,13 @@ Teleport configuration as below:
         second_factor: on
         webauthn:
+          # Required, replace with proxy web address (example.com, mytenant.teleport.sh),
+          #  see `webauthn` Fields Definitions for details.
```
This comment applies to all the changes.

My concern is that once this is copied into a config, the reference to a section in this page is lost. For these comments to be useful once in place, they would have to have some information themselves, or link to the docs page. 
But if that's not a concern to anyone else, my remaining comment is to remove the backticks, since they do nothing in this format.

zmb3 replied:
I agree with the concern. Let's just put a very short explanation in the YAML rather than mention another part of the doc. For example:

```
   # Optionally allow specific MFA device vendors.
   attestation_allowed_cas:
```

stevenGravy replied:
See updates

alexfornuto reviewed (commented):
While we're in this page, we could replace the "Dynamic resources" method to use `tctl edit`.

stevenGravy committed:
Merge branch 'master' into stevenGravy/docs/webauthn

alexfornuto reviewed (changes requested):
Changing my review type since my concerns were corroborated.

stevenGravy committed:
Merge branch 'master' into stevenGravy/docs/webauthn

alexfornuto commented:
@stevenGravy just checking that you're looking into addressing the feedback. I see you rebased the master branch into this PR yesterday.

stevenGravy commented:
> @stevenGravy just checking that you're looking into addressing the feedback. I see you rebased the master branch into this PR yesterday.

Yes I am

stevenGravy committed:
Merge branch 'master' into stevenGravy/docs/webauthn

stevenGravy committed:
Update webauthn definitions

stevenGravy commented:
> Changing my review type since my concerns were corroborated.

See updates pls

alexfornuto commented:
> While we're in this page, we could replace the "Dynamic resources" method to use `tctl edit`.

@stevenGravy do you mind if I take a pass to apply this suggested update?

Commit:
consolidate cap edit options

alexfornuto commented:
@stevenGravy please review 5db9e099288b4c4e7477260aed50a8831d79cf4c, wherein I consolidated the options for editing the `cluster_auth_preference` resource.

Commit:
formatting edits

stevenGravy committed:
Merge branch 'master' into stevenGravy/docs/webauthn
